sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsModelData.cs

```diff
@@ -29,6 +30,14 @@ public partial class DigitalTwinsModelData
         [CodeGenMember("UploadTime")]
         public DateTimeOffset? UploadedOn { get; }
 
+        /// <summary> A language map that contains the localized display names as specified in the model definition. </summary>
+        [CodeGenMember("DisplayName")]
+        public IReadOnlyDictionary<string, string> DisplayNameLanguageMap { get; }
```

Comment:
Map isn't a preferred word for .NET I think.

Reply:
This description comes from swagger, but I can change it to dictionary since we are overriding the name anyways

Reply:
@KrzysztofCwalina your input here would be appreciated.

Reply:
I'll change the property name to be Dictionary instead as well

Reply:
I'm not even sure it is Dictionary. @pakrym do you know?

Reply:
Typically we don't include either Map or Dictionary into property names. What about `LanguageDisplayNames`?

Reply:
Winner, winner, chicken dinner!